### ISSUE TITLE
RGB in nicknames

### ISSUE THREAD

**Issue by ItsHammy** [GeorgH93/MarriageMaster#221]
### Feature request

**Feature description**
Add support for custom colours in 1.16+

**How the feature is useful**
Nicknames currently translate to old style colour codes, which are ugly when using custom nicknames with RGB. Would be nice for the RGB to carry over.

**Comment by thiagogebrimm** [comment]
Yes, this RGB is really needed, sometimes the message is very ugly

**Comment by thiagogebrimm** [comment]
Please?

**Comment by diademiemi** [comment]
This is being discussed in https://github.com/GeorgH93/MarriageMaster/issues/247, just saying "Please?" isn't going speed up development. There is a dev branch with this feature in the works already
